hcl/eval/object_test.go

```diff
@@ -41,8 +45,13 @@ func TestCtyObjectSetAt(t *testing.T) {
 		wantErr error
 	}
 
+	defaultOrigin := eval.Origin{
+		DefinedAt:  "/file.tm",
```

Comment:
DefinedAt is supposed to be a file not a dir ? in that case won't conflict detection fail ? globals defined on the same dir but on different files are supposed to conflict no ?

Reply:
Ah OK I saw you get the Dir(), my bad :laughing: